doc/plutus-core-spec/cardano/builtins1.tex

```diff
@@ -416,10 +416,7 @@ \subsection{Built-in functions}
 \note{Ed25519 signature verification.}
 \label{note:ed25519-signature-verification}
 The \texttt{verifyEd25519Signature}
-function\footnote{\texttt{verifyEd25519Signature} was formerly called
```

Comment:
It might be worth having "(formerly \texttt{verifySignature})" somewhere just in case anyone's been reading a really old version of the document.